client/src/components/StatsList/StatButton/StatButton.scss

```diff
@@ -20,3 +20,20 @@ $stat-btn-text-color: $grey-100;
     background-color: shade($stat-btn-background-color, 20%);
   }
 }
+
+.stat-button--top-trump {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  border-radius: 8px;
+  border: transparent;
+  padding: 5px 16px;
+  text-transform: capitalize;
+  background-color: shade($yellow, 30%);
+  color: $stat-btn-text-color;
+
+  &:hover,
+  &:focus {
+    background-color: shade($stat-btn-background-color, 20%);
+  }
+}
```

Comment:
With the modifier class added conditionally, this can then be updated to:

`.stat-button--top-trump {
  --background-color: #{$stat-btn-background-color--top-trump};

  &:hover,
  &:focus {
    --background-color: #{shade($stat-btn-background-color--top-trump, 20%)};
  }
}`

with `$stat-btn-background-color--top-trump: shade($yellow, 30%);` added at the top alongside the other scss variables

Reply:
Changes have been made to code and shade yellow variable set on line 6 - marked as resolved